actor/v7action/service_instance.go

```diff
@@ -10,6 +10,11 @@ import (
 	"code.cloudfoundry.org/cli/util/railway"
 )
 
+type ServiceInstanceUpdateManagedParams struct {
+	Tags       types.OptionalStringSlice
+	Parameters types.OptionalObject
+}
+
```

Comment:
It wasn't immediate to me why we pulled this out of the `resources` package into the `v7action` package. Looking at the other commands, some implement something similar, some use the resource struct like the one we were using before. I'm inclined to believe it should as it was implemented before 🤔

Reply:
It’s needed for the next story (service plans, because the command layer will specify a plan name, while the resource only has a GUID field). I agree that it’s not technically necessary here. At one point I thought that it was, and once I realised it was only needed for the next flag I decided to leave it in place.